RedHat-UX/red-hat-design-system
# fix(accordion): correct header text font weight

## What I did

1. Corrected header text font-weight


## Testing Instructions

1. View Deploy Preview, Safari, Firefox, and Chrome should have the same font-weight.

## Notes to Reviewers
